imgix/constants.py

```diff
@@ -5,3 +5,20 @@
             r'(?:[a-z\d](?:\-(?=\-*[a-z\d])|[a-z]|\d){0,62}\.)'
             r'[a-z\d]{1,63}$'
         )
+
+
+def target_widths():
```

Comment:
Since this is only used in one class, generally it would be placed in that class (in this case UrlBuilder), with the constants (`INCREMENT_PERCENTAGE`, even a `MIN_SIZE = 100` to use for the initial value of prev) pulled out to the top of the class and probably given more explicit names (`SRCSET_INCREMENT_PERCENTAGE` for example)